cfme/configure/access_control/__init__.py

```diff
@@ -1115,7 +1119,7 @@ def copy(self, name=None):
         view.toolbar.configuration.item_select('Copy this Role to a new Role')
         view = self.create_view(AddRoleView, wait=10)  # implicit assert
         new_role = self.parent.instantiate(name=name)
```

Comment:
This should be accomplished by setting `vm_restriction` on `new_role`, and referencing it when filling the copy form.

Otherwise the role instance generated by the copy won't have the vm_restriction matching what it should be in the UI.

Reply:
:D nice catch... moved arg name similar to attribute name :+1: